Apps/W1/Shopify/app/src/Products/Codeunits/ShpfyProductEvents.Codeunit.al

```diff
@@ -347,4 +347,29 @@ codeunit 30177 "Shpfy Product Events"
     internal procedure OnBeforeSendUpdateShopifyProductVariant(ShopifyShop: Record "Shpfy Shop"; var ShopifyVariant: Record "Shpfy Variant"; xShopifyVariant: Record "Shpfy Variant")
     begin
     end;
-}
\ No newline at end of file
+
+    [IntegrationEvent(false, false)]
+    /// <summary> 
+    /// Raised Before Modify on Items.
+    /// </summary>
+    /// <param name="ShopifyShop">Parameter of type Record "Shopify Shop".</param>
+    /// <param name="ShopifyProduct">Parameter of type Record "Shpfy Product".</param>
+    /// <param name="Item">Parameter of type Record "Item".</param>
+    /// <param name="IsModified">Parameter of type Boolean.</param>
+    internal procedure OnDoUpdateItemBeforeModify(var ShopifyShop: Record "Shpfy Shop"; var ShopifyProduct: Record "Shpfy Product"; var Item: Record Item; IsModified: Boolean);
```

Comment:
@viktorglemne I think you forgot to set the parameter `IsModified` as `var`, that this parameter can be updated inside an event.
@AndreiPanko

Reply:
Yes that was an miss! @pri-kise its now updated